sdk/core/azure-core/inc/azure/core/http/curl/curl connection_pool.hpp

```diff
@@ -26,83 +26,161 @@ namespace Azure { namespace Core { namespace Test {
 #endif
 
 namespace Azure { namespace Core { namespace Http {
+  /**
+   * @brief The available options to set libcurl ssl options.
+   *
+   * @remark The SDK will map the enum option to libcurl's specific option. See more info here:
+   * https://curl.haxx.se/libcurl/c/CURLOPT_SSL_OPTIONS.html
+   *
+   */
+  struct CurlTransportSSLOptions
+  {
+    bool AllowBeast = false;
+    bool NoRevoke = false;
+    bool NoPartialchain = false;
```

Comment:
These double negatives are hard to reason about.

Reply:
That said, I understand we probably want `false` for all of these as the default. Do we really want `NoPartialChain` as false by default? I would have thought the default would be `PartialChain = false`.

Reply:
We should not change the original names or default values